src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/payment/PaymentDto.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.payment;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee.FeeDto;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true, value = {"payment_group_reference"})
+@JsonInclude(NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@Data
+public class PaymentDto {
+
+    private String id;
+
+    private BigDecimal amount;
+
+    private String description;
+
+    private String reference;
+
+    @JsonProperty("service_name")
+    private Service service;
+
+    @JsonProperty("date_created")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ", timezone = "GMT")
+    private Date dateCreated;
+
+    @JsonProperty("date_updated")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ", timezone = "GMT")
+    private Date dateUpdated;
+
+    private String currency;
+
+    @JsonProperty("ccd_case_number")
+    private String ccdCaseNumber;
+
+    private String channel;
+
+    private String method;
+
+    @JsonProperty("external_provider")
+    private String externalProvider;
+
+    @JsonProperty("external_reference")
+    private String externalReference;
+
+    private String siteId;
```

Comment:
@tarun-palisetty do we need to update our **PaymentDto** based on Yashar's latest comments?

- For online card payments, we just need to pass `case_type` instead of `site_id` and `service`.
- For PBA, we send `case_type` instead of `site_id` (service still required).

Reply:
Removed siteId